### PR TITLE
add gh action based dev pipeline.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
This is an attempt at a like-for-like pipeline that has the same functionality (and shares sh code with) our existing dev pipeline.

**Review by Fosol — APPROVED** [review]
Can you provide some additional information like a README for this?  It looks like you're building and deploying from GitHub directly to OpenShift.  Does this mean the Jenkins pipeline is no longer used?

File: .github/workflows/dev-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+          ./openshift/4.0/player.sh deploy api $DESTINATION -apply
+          ./openshift/4.0/player.sh deploy app $DESTINATION -apply
+  notify:
+    needs: deploy
+    if: always()
+    name: notify-teams
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Log in to OpenShift
```
Comment (asanchezr) [inline .github/workflows/dev-pipeline.yml:85]: Not sure the name is appropriate here

**Review by asanchezr — APPROVED** [review]
I love this direction. When I created the sh scripts that was the end goal - to get rid of Jenkins and replace with GH actions.

**Comment by asanchezr** [comment]
My only comment with this PR (and the others currently open) is that we are still seeing GH action running for Node 10. Wasn't expecting to see that once the PR to update to Node 14 got merged - ideas?

image

**Pull request closed without merging**